course-definition.yml

```diff
@@ -344,7 +344,9 @@ stages:
 
       The command (`custom_exe_1234`) in the example above will be present in `PATH` and will be an executable file.
 
-      The tester will check if your shell correctly executes the given command and prints the output.
+      You don’t need to print the details from the example; your task is simply to execute the specified command.
```

Comment:
The command will print information about the arguments it was passed along with a random "program signature". The tester will verify that your program executes the given command and prints the output.

Reply:
For Notes: 

> The output in the example ("Program was passed N args...") comes from the executable, it's not something you need to implement manually.